src/index.js

```diff
@@ -121,3 +115,6 @@ module.exports = window.AFRAME = {
   utils: utils,
   version: pkg.version
 };
+
+document.dispatchEvent(new window.Event('aframe-ready'));
```

Comment:
Why is this necessary?

Reply:
The bootstrap sequence in Cordova goes like this:

1. Wait for `deviceready` event
2. Inject `aframe-master.js` script tag
3. Wait for `aframe-master.js` to inject `aframe.css` and `rStats.css` link tags
4. Insert `cordova-aframe.css` link tag

The `aframe-ready` is how I know that A-Frame is done injecting its css tags.

What else can I do to wait for A-Frame to finish loading?

Reply:
Injecting your css after A-Frame should be enough unless cordova operates differently. I don't think you need to wait or an extra event. Loading them in order should work e.g:

```
var el = document.createElement('script');
el.src = 'aframe-master.js';
el.async = false;
document.head.appendChild(el);

var link  = document.createElement('link');
link.rel  = 'stylesheet';
link.type = 'text/css';
link.href = 'http://yoururl.com/style.css';
document.head.appendChild(link);
```

Reply:
A-Frame's css doesn't get injected until the next tick. [This glitch](https://glitch.com/~turquoise-grand-eocursor) shows the situation.

Reply:
Load the styles from a JS file so they're both sync. Your example modified:

https://glitch.com/edit/#!/tundra-cotton?path=cordova-aframe.js:10:22

Reply:
We can remove this. Depending on the asset base URL and polyfillConfig strategies you're comfortable with, I may need to maintain a custom build anyway. If that happens, I will keep this event because your solution adds a layer of indirection that feels unnecessarily complex for me as a Cordova user. I understand you want to minimize core changes though.